code/website/index.html

```diff
@@ -1,22 +1,47 @@
 <h2>Drasil - Generate All the Things!</h2>
+<p id="center"><a href="https://github.com/JacquesCarette/Drasil">Link to GitHub repository</a></p>
+<hr>
+<br />
```

Comment:
Pedantic:
`br` is a [void element](https://www.w3.org/TR/2011/WD-html-markup-20110405/syntax.html#syntax-elements) and thus it is more correct (in my opinion) as `<br>` as opposed to `<br/>`.

Reply:
I can change them `<br>`.